examples/hip_modules_example/BUILD.bazel

```
@@ -0,0 +1,66 @@
+load(
+    "@rules_ll//ll:defs.bzl",
+    "ll_binary",
+    "ll_library",
+)
+
+cc_binary(
+    name = "cc",
+    srcs = ["main.cpp"],
+    copts = [
+        "-v",
+    ],
+    linkopts =
+        [
+            "--verbose",
+        ],
+)
+
+ll_library(
+    name = "hip",
+    srcs = ["hip.cpp"],
+    compilation_mode = "hip_nvptx",
+    compile_flags = [
+        "--std=c++20",
+        "--offload-arch=sm_61",
+        "-Wno-deprecated",
+    ],
```

```suggestion
    compile_flags = OFFLOAD_ALL_NVPTX + [
        "--std=c++20",
        "-Wno-deprecated",
    ],
```
so that this works on all machines.